scripts/cloud_add_wrapper.sh

```diff
@@ -6,6 +6,20 @@ team_id="YOUR_TEAM_ID"
 role_name="securestate_role"
 while read account_name account_id environment profile; do
 	echo "creating role for account: $account_id"
+	accounts=$(coreo cloud list --json --team-id $team_id | jq -r .[].name)
+	is_exist=false
+	while read account
+	do
+		if [ "$account" = "$account_name" ]; then
+			is_exist=true
+			break
+		fi
+	done <<< "$accounts"
+	if [ $is_exist = true ]; then
+		echo "Cloud account with name $account_name already exist, skip for this account"
+		continue
+	fi
```

Comment:
If the cloud account exists, I only want to skip the coreo cloud add. We should always do the coreo event setup.